aspnetcore/grpc/basics.md

```diff
@@ -9,53 +9,60 @@ uid: grpc/basics
 
-## `.proto` file
+## proto file
 
-gRPC uses a contract-first approach to API development. Protocol buffers (protobuf) are used as the Interface Design Language (IDL) by default. As a result, `.proto` file serve as the definition of the gRPC service and messages sent between clients and servers. For more information on the syntax of protobuf files, visit the [official documentation (protobuf)](https://developers.google.com/protocol-buffers/docs/proto3)).
+gRPC uses a contract-first approach to API development. Protocol buffers (protobuf) are used as the Interface Design Language (IDL) by default. The *.proto* file contains:
 
-For example, the `greet.proto` file defines a `Greeter` service which defines a `SayHello` call that sends a `HelloRequest` message and receives a `HelloResponse` message:
+* The definition of the gRPC service.
+* The  messages sent between clients and servers.
+
+For more information on the syntax of protobuf files, see the [official documentation (protobuf)](https://developers.google.com/protocol-buffers/docs/proto3)).
+
+For example, consider the *greet.proto* file used in [Get started with gRPC service](xref:tutorials/grpc/grpc-start):
+
+* Defines a `Greeter` service.
+* The `Greeter` service defines a `SayHello` call.
+* `SayHello` sends a `HelloRequest` message and receives a `HelloResponse` message:
 
 [!code-proto[](~/tutorials/grpc/grpc-start/samples/GrpcStart/Protos/greet.proto)]
 
-## Add a `.proto` file to your C# app
+## Add a .proto file to a C# app
 
-The `.proto` file is included in your project by adding it to the `<Protobuf>` item group:
+The *.proto* file is included in a project by adding it to the `<Protobuf>` item group:
 
-[!code-xml[](~/tutorials/grpc/grpc-start/samples/GrpcStart/GrpcGreeter.Server/GrpcGreeter.Server.csproj?highlight=8)]
+[!code-xml[](~/tutorials/grpc/grpc-start/samples/GrpcStart/GrpcGreeter.Server/GrpcGreeter.Server.csproj?highlight=2&range=7-10)]
 
-## C# Tooling support for `.proto` files
+## C# Tooling support for .proto files
 
-The tooling package [Grpc.Tools](https://www.nuget.org/packages/Grpc.Tools/) is required to generate the C# assets from `.proto` files. The assets are generated on a as-needed basis every time the project is built. This dependency is required by both the server and client projects and can be added by using the Package Manager in Visual Studio or adding a `<PackageReference>` to your project file:
+The tooling package [Grpc.Tools](https://www.nuget.org/packages/Grpc.Tools/) is required to generate the C# assets from *.proto* files. The assets are generated on a as-needed basis each time the project is built. This dependency is required by both the server and client projects. `Grpc.Tools` can be added by using the Package Manager in Visual Studio or adding a `<PackageReference>` to the project file:
```

Comment:
I think it is important to mention that the generated files are not added to the project or checked into source control.

Is there a standard way of saying they are a build artifact that lives in the obj directory?